test/e2e/useraccount_test.go

```diff
@@ -109,6 +112,24 @@ func TestUserAccount(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("delete_useraccount_ok", func(t *testing.T) {
+		lucyAcc := &toolchainv1alpha1.UserAccount{}
+		err := client.Get(context.TODO(), types.NamespacedName{Name: lucyUserAcc.Name, Namespace: namespace}, lucyAcc)
+		assert.NoError(t, err)
+
+		err = client.Delete(context.TODO(), lucyAcc)
+		assert.NoError(t, err)
+
+		err = waitForDeletedUserAccount(t, client, lucyAcc.Name, namespace)
+		assert.NoError(t, err)
+
+		err = waitForDeletedUser(t, client, lucyAcc.Name)
+		assert.NoError(t, err)
+
+		err = waitForDeletedIdentity(t, client, lucyAcc.Name)
```

Comment:
Can you please verify that `useridentitymapping` is also deleting? https://docs.openshift.com/container-platform/3.7/rest_api/apis-user.openshift.io/v1.UserIdentityMapping.html#Post-apis-user.openshift.io-v1-useridentitymappings

Reply:
Looks like i cannot get the useridentitymapping resource. I see the following error: 
```
tkurian@localhost host-operator]$ oc get useridentitymapping
Error from server (MethodNotAllowed): the server does not allow this method on the requested resource
```
It is also outlined in this defect: https://bugzilla.redhat.com/show_bug.cgi?id=1684311

But as you suggested, i have tried adding and removing and re-adding the same userAccount and nothing fails!

Reply:
@dipak-pawar useridentitymapping is a special resource. It's just a REST API for CREATE. In reality it modifies User and Identity. It doesn't have it's own etcd representation (nor get, watch, etc API). So, if both User and Identity are deleted then there is no any UserIdentityMapping either.